files/models.py

```
@@ -2164,3 +2165,98 @@ def media_pre_save(sender, instance, **kwargs):
+
+    class Meta:
+        ordering = ["-start_date"]
+        verbose_name = "Featured Video Schedule"
+        verbose_name_plural = "Featured Video Schedules"
+
+    def __str__(self):
+        return f"{self.media.title} ({self.start_date.strftime('%Y-%m-%d')})"
+
+
+@receiver(pre_save, sender=Media)
+def track_featured_change(sender, instance, **kwargs):
```

Ruff (0.14.8)

Unused function argument: `sender`

(ARG001)